content/en/docs/components/central-dash/overview.md

```diff
@@ -64,19 +72,19 @@ guide to
 
 ## Using kubectl and port-forwarding
 
-If you didn't configure Kubeflow to integrate with an identity provider 
+If you didn't configure Kubeflow to integrate with an identity provider
 then you can port-forward directly to the Istio gateway.
 
 Port-forwarding typically does not work if any of the following are true:
 
-  * You've deployed Kubeflow on GCP using the 
-    [GCP deployment UI](/docs/gke/deploy/deploy-ui/) or the default settings 
+  * You've deployed Kubeflow on GCP using the
+    [GCP deployment UI](/docs/gke/deploy/deploy-ui/) or the default settings
```

Comment:
Would you like to remove the GCP deployment UI approach? We have changed to use CLI deployment only